docs/hub/datasets-libraries.md

```diff
@@ -4,24 +4,52 @@ The Datasets Hub has support for several libraries in the Open Source ecosystem.
-| [Polars](./datasets-polars)         | A DataFrame library on top of an OLAP query engine.                                                                            | ✅                | ✅          |
-| [PyArrow](./datasets-pyarrow)       | Apache Arrow is a columnar format and a toolbox for fast data interchange and in-memory analytics.                             | ✅                | ✅          |
-| [Spark](./datasets-spark)           | Real-time, large-scale data processing tool in a distributed environment.                                                      | ✅                | ✅          |
-| [WebDataset](./datasets-webdataset) | Library to write I/O pipelines for large datasets.                                                                             | ✅                | ❌          |
+| [DuckDB](./datasets-duckdb)         | In-process SQL OLAP database management system.                                                                                | ✅ +s             | ❌          |
+| [Embedding Atlas](./datasets-embedding-atlas) | Interactive visualization and exploration tool for large embeddings.                                                 | ✅ +s             | ❌          |
+| [Fenic](./datasets-fenic)           | PySpark-inspired DataFrame framework for building production AI and agentic applications.                                      | ✅ +s             | ❌          |
+| [FiftyOne](./datasets-fiftyone)     | FiftyOne is a library for curation and visualization of image, video, and 3D data.                                             | ✅ +s             | ✅          |
+| [Pandas](./datasets-pandas)         | Python data analysis toolkit.                                                                                                  | ✅                | ✅ +p*      |
+| [Polars](./datasets-polars)         | A DataFrame library on top of an OLAP query engine.                                                                            | ✅ +s             | ✅          |
+| [PyArrow](./datasets-pyarrow)       | Apache Arrow is a columnar format and a toolbox for fast data interchange and in-memory analytics.                             | ✅ +s             | ✅ +p*      |
+| [Spark](./datasets-spark)           | Real-time, large-scale data processing tool in a distributed environment.                                                      | ✅ +s             | ✅ +s +p    |
+| [WebDataset](./datasets-webdataset) | Library to write I/O pipelines for large datasets.                                                                             | ✅ +s             | ❌          |
+
+_+s: Supports Streaming_
+_+p: Writes optimized Parquet files_
+_+p*: Requires passing extra arguments to write optimized Parquet files_
+
+### Streaming
+
+Dataset streaming allows to iterate on a dataset on Hugging Face progressively without having to download it completely.
+It saves disk space and download time.
```

Comment:
curious to know, how does it save download time?

Reply:
Maybe it's a bit unclear. You don't need to spend time downloading data in advance since it streams in parallel to the cpu / gpu workload. Let me add general clarifications here:

```suggestion
It saves disk space since the data is never materialized on disk. It saves memory since only a small portion of the dataset is used at a time. And it saves time, since there is no need to download data in advance prior to the actual CPU or GPU workload.
```